### DISCUSSION TITLE
O2M interface item display improvement

### DISCUSSION THREAD

**Discussion by goggi** [directus/directus#3034] [Feature Requests]
This is a minor ux improvement suggestion for the O2M interface. 

I'm displaying 3 columns in the O2M field and there is space for all of them to be visible. But it seems like it's using a fixed size for each column.  

![image](https://user-images.githubusercontent.com/1946339/96719514-c3c80b00-13a9-11eb-8649-7c1270bd9a61.png)

Would be nice if all the fields are fully visible if there is space. 

![image](https://user-images.githubusercontent.com/1946339/96719583-dd695280-13a9-11eb-90e4-07d3b1f32ba5.png)

---

### Project details
Directus Version: 9.0.0-rc.1
Environment: production
OS: Linux 4.15.0-88-generic
Node: 14.7.0

**Comment by benhaynes** [comment]
The same applies to the table layout too, ideally we add "right-click" to the column headers, and there are resize options for manual (px) or auto (tries to fit content). We should also discuss how we want to handle min-width, max-width, and recommended width (eg: for the status dot or icons).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
